streams: Add stream traffic data to `Stream` objects. (#26398)
zulip/zulip: api_docs/changelog.md

## Changes in Zulip 8.0

**Feature level 199**

* [`POST /register`](/api/register-queue), [`GET /events`][/api/get-events],
  [`GET /streams`](/api/get-streams),

>   [`GET /streams/{stream_id}`](/api/get-stream-by-id): Include stream traffic
>   data using the `stream_weekly_traffic` field in Stream objects.

Reviewer: Here's a slight rearrangement:
```suggestion
  [`GET /streams/{stream_id}`](/api/get-stream-by-id): Stream objects
  now include a `stream_weekly_traffic` field indicating the stream's level of traffic.
```

I like starting with "stream objects" because it scopes where the change occurred. That helps a reader be able to quickly stop reading and move to the next item if they're looking for changes that could affect some particular thing they're working on.
